notebooks/ch14/bias_modeling.md

```diff
@@ -81,6 +81,45 @@ The second term in this expression, $ \text{Var}(f_\hat{\theta}(z)) $, represent
 
 The third and final term in this expression, $ \text{Var}(\epsilon) $, represents the irreducible error or the noise in the data generation and collection process. This term is small when the data generation and collection process is precise or has low variation. This term is large when the data contain large amounts of noise.
 
+To begin the decomposition, we start with the mean squared error
+
+$$\mathbb{E}[(\gamma - f_{\hat{\theta}}(z))^2]$$
+
+and square the inside term and apply linearity of expectation:
```

Comment:
Just for aesthetic and consistency purposes, let's always capitalize the first word of a line: `and` -> `And`. Also, we don't `square` the inside term—we're expanding the square.

Reply:
fixed